src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -197,10 +276,9 @@ tnsr::Ij<tt::remove_cvref_wrap_t<T>, 3, Frame::NoFrame> Frustum::jacobian(
       orientation_of_frustum_.inverse_map()(Direction<3>::upper_xi());
   const size_t mapped_dim_0 = orientation_of_frustum_.inverse_map()(0);
   jacobian_matrix.get(mapped_dim_0, 0) =
-      delta_x_xi_ + delta_x_xi_zeta_ * cap_zeta;
-  if (with_equiangular_map_) {
-    jacobian_matrix.get(mapped_dim_0, 0) *= cap_xi_deriv;
-  }
+      (delta_x_xi_ + delta_x_xi_zeta_ * cap_zeta) *
+      (with_equiangular_map_ ? cap_xi_deriv
+                             : make_with_value<ReturnType>(xi, 1.0));
```

Comment:
I suggest to keep the `if (with_equiangular_map)` instead of the new `make_with_value` (it introduces an unnecessary allocation and pointwise multiplication by one). This occurs a few more times below, also with `with_projective_map_`.